src/test/kotlin/com/atlassian/performance/tools/report/api/result/CohortResultTest.kt

```diff
@@ -57,4 +56,40 @@ class CohortResultTest {
         ).build()
         assertThat(actualLatest, not(equalTo(straggler)))
     }
+
+    @Test
+    fun shouldNotBeAffectedByOutliers() {
+        val anticipatedLoad = VirtualUserLoad(
+            virtualUsers = 20,
+            flat = Duration.ofMinutes(10)
+        )
+        val criteria = PerformanceCriteria(
+            mapOf(
+                CREATE_ISSUE_SUBMIT to Criteria(
+                    toleranceRatio = 0.04F,
+                    minimumSampleSize = 12,
+                    acceptableErrorCount = Int.MAX_VALUE
+                )
+            ),
+            virtualUserLoad = anticipatedLoad,
+            maxVirtualUsersImbalance = 18,
+            nodes = 2
+        )
+        val resultsPath = Paths.get("JIRA-JPT-JOB1-9799")
```

Comment:
Is this path element "JIRA-JPT-JOB1-9799" somehow fixed?  It looks like containing hardcoded build number.

Reply:
OK. I see You added resources.

Reply:
By the way, WDYT about storing resources for this test under name (used within "path") that comes from issue number or the problem You solve itself (something like 'outlier trim test data' or whatever similar) ?

Reply:
> OK. I see You added resources.
Yeah, it's consistent with resources added for other tests.

Reply:
> By the way, WDYT about storing resources for this test under name (used within "path") that comes from issue number or the problem You solve itself (something like 'outlier trim test data' or whatever similar) ?

I will rename it to `CREATE-ISSUE-WITH-OUTLIERS`